cps2pc

```diff
@@ -0,0 +1,206 @@
+        pc_dict['Libs'] += get_link_features(component.link_features, platform)
+        pc_dict['Libs'] += get_cpp_runtime_vendor(package)
+        try:
+            pc_dict.update(handle_kind(component.kind, component.location))
+        except Exception as e:
+            print 'Skipping %s: %s kind is not supported by pkgconfig' % (component_name, e)
+            continue
+        pc_dict['Cflags'] = []
+        pc_dict['Cflags'] += list(component.compile_flags[None])
+        pc_dict['Cflags'] += ["-I%s" % x for x in component.includes[None]]
+        pc_dict['Cflags'] += get_definitions_flags(component.definitions[None], compiler)
+        pc_dict['Cflags'] += get_compile_features(component.compile_features, compiler)
+        pc_dict['Requires'] = [x.split(':')[-1] for x in component.requires]  # Remove possible package name
+        remove_empty_keys(pc_dict)
+        filename = os.path.join(output_dir, component_name + ".pc")
+        write_pc(pc_dict, filename, prefix)
+
+
+def check_compiler_support(compiler):
+    if compiler not in _supported_compilers:
+        raise Exception('Compiler %s not supported. Check supported compiler with --supported-compiler' % compiler)
+
+
+class PrintMessageActionAction(argparse.Action):
+    def __init__(self,
+                 option_strings,
+                 message,
+                 dest=argparse.SUPPRESS,
+                 default=argparse.SUPPRESS,
+                 help=None):
+        self._message = message
+        super(PrintMessageActionAction, self).__init__(
+            option_strings=option_strings,
+            dest=dest,
+            default=default,
+            nargs=0,
+            help=help)
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        print(self._message)
+        parser.exit()
+
+#------------------------------------------------------------------------------
+def main(args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('input', metavar='CPS_FILE', type=str,
+                        help='Input CPS file')
+    parser.add_argument('output', metavar='OUTPUT_DIRECTORY', type=str,
+                        help='Output directory for pkgconf files')
+    parser.add_argument('--compiler', type=str,
+                        default=_supported_compilers[0],
+                        help='Targeted compiler')
+    parser.add_argument('--prefix', type=str,
+                        default=_default_prefix,
+                        help='Use given prefix instead of %s'%_default_prefix)
+    parser.add_argument('--supported-compilers',
+                        message = "Supported compilers: %s" % ', '.join(_supported_compilers),
+                        action=PrintMessageActionAction,
+                        help='List supported compilers')
+
+    args = parser.parse_args(args)
+    package = cps.read(args.input, canonicalize=True)
+    # Fail-safe function to get an easy to understand error message.
```

Comment:
A blank line before here, and before `write_targets_config`, would be nice :slightly_smiling_face:.

Reply:
A blank line has been added.